qutebrowser/browser/qutescheme.py

```diff
@@ -201,6 +201,31 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+def _tab_fields_to_tabs_page_info(fields):
+    return (fields['title'], fields['current_url'])
+
+
+@add_handler('tabs')
+def qute_tabs(_url):
+    """Handler for qute://tabs. Display information about all open tabs."""
+    tabs = {}
```

Comment:
You can use a `collections.defaultdict(list)` here, then you don't need to set it to `[]`.